productPage/singleProduct.css

```diff
@@ -0,0 +1,31 @@
+.product-description {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    gap: var(--size-08);
+    margin-top: var(--size-12);
+}
+
+.product-details {
+    align-items: center;
+    display: flex;
+    flex-flow: row wrap;
+}
+
+.product-details img {
+    width: 50%;
+    height: 80%;
```

Comment:
We should not use height for images ideally.

Reply:
Yeah just to restrict it to the same view-page I have done that.